### ISSUE TITLE
CSS missing semicolon should throw error?

### ISSUE THREAD

**Issue by noyobo** [parcel-bundler/lightningcss#398]
```css
.less {
  color: red
  background-color: green;
}
```

[playground](https://lightningcss.dev/playground/index.html#%7B%22minify%22%3Atrue%2C%22nesting%22%3Atrue%2C%22customMedia%22%3Atrue%2C%22cssModules%22%3Afalse%2C%22analyzeDependencies%22%3Afalse%2C%22targets%22%3A%7B%22chrome%22%3A6225920%7D%2C%22source%22%3A%22.less%20%7B%5Cn%20%20color%3A%20red%5Cn%20%20background-color%3A%20green%3B%5Cn%7D%22%2C%22visitorEnabled%22%3Atrue%2C%22visitor%22%3A%22%7B%5Cn%20%20Color(color)%20%7B%5Cn%20%20%20%20if%20(color.type%20%3D%3D%3D%20'rgb')%20%7B%5Cn%20%20%20%20%20%20color.g%20%3D%200%3B%5Cn%20%20%20%20%20%20return%20color%3B%5Cn%20%20%20%20%7D%5Cn%20%20%7D%5Cn%7D%22%2C%22unusedSymbols%22%3A%5B%5D%2C%22version%22%3A%22local%22%7D)

**Comment by devongovett** [comment]
The way that parses is as a single property `color`, with the value `red background-color: green`.

<img width="271" alt="image" src="https://user-images.githubusercontent.com/19409/212358640-e08deafa-c05e-4ac5-90ad-13e21b67d235.png">

In this case, the value isn't valid for the `color` property, but if you were using a custom property like `--color` it would be. So it's not really a parse error, just that the value isn't valid.

**Comment by noyobo** [comment]
Yes, but such syntax (line breaks) is not a CSS declaration. before use lightningcss, I need to lint through other tools, such as prettier or stylelint.

<img width="959" alt="image" src="https://user-images.githubusercontent.com/1292082/212458315-441e19da-76f7-44bd-ba87-2ebb616f0cf7.png">

**Comment by noyobo** [comment]
Are there plans to fix this? Because when typing dynamically, the developer might forget to enter the semicolon, so the line breaks.

This was not the developer's intention and does not make sense.

Can you report an error? @devongovett
